pkg/api/authz.go

```diff
@@ -341,3 +339,13 @@ func anonymousPolicyExists(config *config.AccessControlConfig) bool {
 
 	return false
 }
+
+func isExtensionRequest(request *http.Request) bool {
+	if request.RequestURI == fmt.Sprintf("%s%s", constants.RoutePrefix, constants.ExtCatalogPrefix) ||
```

Comment:
I suggested some changes to this logic in my latest review of https://github.com/project-zot/zot/pull/1317.
But otherwise the proposal looks good to me.